### PR TITLE
Fix: The Last Blade

### PR DESCRIPTION
**Pull request by ranplayer**

### CODE REVIEW

**Comment by R1kaB3rN** [comment]
I read the upstream report and this looks to be graphics related. 

Is there any reason why you haven't reported this upstream in DXVK? Because having this fixed in DXVK would be more preferable than a fix in this project.

**Comment by R1kaB3rN** [comment]
Sorry, never mind. I was looking into this issue and saw https://github.com/doitsujin/dxvk/issues/4095, so it appears to be a problem in wine.

File: gamefixes-steam/465840.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+""" 
+The Last Blade
+"""
+#pylint: disable=C0103
+
+from protonfixes import util
+
+def main():
+    util.protontricks('d3dcompiler_46')
+
```
Comment (R1kaB3rN) [inline gamefixes-steam/465840.py:10]: After this trailing new line is deleted, I'll merge this ASAP.

Comment (ranplayer) [reply gamefixes-steam/465840.py]: done

**Pull request merged**